From 7e02117722915b821afe224cbdf7c53209877bcb Mon Sep 17 00:00:00 2001
From: George Beshers <gbeshers@redhat.com>
Date: Thu, 18 Dec 2014 03:07:48 -0500
Subject: [x86] uv: make kdump default action for 'power nmi'

Message-id: <20141218030748.21506.16098.sendpatchset@sgi-xe250-01.lab.bos.redhat.com>
Patchwork-id: 102231
O-Subject: [RHEL7.1 PATCH] UV: make kdump default action for 'power nmi'
Bugzilla: 1175560
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

BZ1175560: UV: make kdump default action for 'power nmi'
First Post

BrewBuild: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=8397923

Change the default 'power nmi' action from kdb to kdump because
that is much more likely to be useful at a customer site.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/platform/uv/uv_nmi.c b/arch/x86/platform/uv/uv_nmi.c
index 4a2b7f7..c6fd363 100644
--- a/arch/x86/platform/uv/uv_nmi.c
+++ b/arch/x86/platform/uv/uv_nmi.c
@@ -151,7 +151,7 @@ module_param_named(retry_count, uv_nmi_retry_count, int, 0644);
  *  "kdb" - enter KDB (default)
  *  "kgdb" - enter KGDB
  */
-static char uv_nmi_action[8] = "kdb";
+static char uv_nmi_action[8] = "kdump";
 module_param_string(action, uv_nmi_action, sizeof(uv_nmi_action), 0644);
 
 static inline bool uv_nmi_action_is(const char *action)
-- 
1.7.1